packages/ts/react-crud/src/autoform-field.tsx

```diff
@@ -93,6 +98,9 @@ export type AutoFormFieldProps = CheckboxProps &
   TimePickerProps;
 
 export function AutoFormField(props: AutoFormFieldProps): JSX.Element | null {
+  if (props.options?.label) {
+    props.propertyInfo.humanReadableName = props.options.label;
+  }
```

Comment:
I think that this is controversial, especially given that we already know that `options` will also replace `customFields`. In fact, doing both in the same PR would make it easier to think about the best approach for merging default values with the ones coming from options.